packages/amplify-provider-awscloudformation/src/upload-appsync-files.ts

```diff
@@ -189,9 +189,9 @@ async function uploadAppSyncFiles(context, resourcesToUpdate, allResources, opti
     if (!fs.existsSync(resourceBuildDir)) {
       return;
     }
-    const spinner = new ora('Uploading files...');
+    const spinner = ora('Uploading files...');
```

Comment:
spinner needs to be in `try...finally` to make sure it will be destroyed and not causing a hang for the process